src/Program.cs

```diff
@@ -40,9 +40,9 @@ private static async Task<int> Main(string[] args)
         public static async Task<int> Run(
             string? workspace,
             bool folder,
-            bool whitespace,
-            string? style,
-            string? analyzers,
+            bool fixWhitespace,
```

Comment:
so if these don't match things just silently stop working?

Reply:
I am working on a test that will help us stay correct.

Reply:
Added test and delegate.